.env.example

```diff
@@ -141,6 +141,10 @@ QUEUE_NAMESPACE=DEV
 #AWS_SQS_SECRET_ACCESS_KEY=
 #AWS_SQS_DEFAULT_REGION=
 
+# GOOGLE CLOUD PUBSUB creentials
+#GOOGLECLOUD_PUBSUB_KEY_FILENAME=
```

Comment:
Can we make this consistent either `GOOGLE_CLOUD` or `GOOGLECLOUD`?

Reply:
Yea that's just wrong - my bad. Should have been GOOGLE_CLOUD (and is consistent with the storage as well)